app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -102,3 +102,52 @@ img {
   height: 25vh;
   background-color: rgba(255,255,255,0.5);
 }
+
+.apartment-info {
+  background-color: #c5499b;
+  box-sizing: border-box;
```

Comment:
Properties should be ordered background-color, border-radius, box-sizing, color, display, font-family, font-size, margin, padding, text-align, width